<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PizzaShopMenu</class>
 <widget class="QMainWindow" name="PizzaShopMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PizzaShopMenu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(200, 194, 22);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 20pt &quot;Tahoma&quot;;</string>
    </property>
    <property name="text">
     <string>JOE'S PIZZA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CalculateButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>550</y>
      <width>269</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>370</width>
      <height>490</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="FoodLogo.qrc">:/img/Yellow Simple Pizza Member Business Card (1).jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>271</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 700 12pt &quot;Tahoma&quot;;</string>
       </property>
       <property name="text">
        <string>TOTAL COST</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pizzaCost">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>141</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 700 12pt &quot;Tahoma&quot;;</string>
       </property>
       <property name="text">
        <string>CRUST TYPE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ThinCrust">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Thin Crust </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ThickCrust">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Thick Crust </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>171</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="DeliveryMethod">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 700 12pt &quot;Tahoma&quot;;</string>
       </property>
       <property name="text">
        <string>DELIVERY METHOD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="HomeDelivery">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Home Delivery</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="TakeOut">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Take Out</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Dinein">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Dine In</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>81</y>
      <width>171</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 700 12pt &quot;Tahoma&quot;;
background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>PIZZA SIZE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="pizzaCombo">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>221</y>
      <width>171</width>
      <height>149</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 700 12pt &quot;Tahoma&quot;;</string>
       </property>
       <property name="text">
        <string>TOPPINGS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="Sausage">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Sausage</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="Pepperoni">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pepperoni</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ExtraCheese">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Extra Cheese</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="Bacon">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Bacon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="Mushrooms">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Mushrooms</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="Onions">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Onions</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>label</zorder>
   <zorder>label_6</zorder>
   <zorder>Dinein</zorder>
   <zorder>Takeout</zorder>
   <zorder>label_5</zorder>
   <zorder>HomeDelivery</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>pizzaCombo</zorder>
   <zorder>ThinCrust</zorder>
   <zorder>ThickCrust</zorder>
   <zorder>Pepperoni</zorder>
   <zorder>Mushrooms</zorder>
   <zorder>Onions</zorder>
   <zorder>Sausage</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label_6</zorder>
   <zorder>pizzaCost</zorder>
   <zorder>ExtraCheese</zorder>
   <zorder>label</zorder>
   <zorder>CalculateButton</zorder>
   <zorder>DeliveryMethod</zorder>
   <zorder>HomeDelivery</zorder>
   <zorder>TakeOut</zorder>
   <zorder>Dinein</zorder>
   <zorder>label_5</zorder>
  </widget>
 </widget>
 <resources>
  <include location="FoodLogo.qrc"/>
 </resources>
 <connections/>
</ui>
